applications/GeoMechanicsApplication/tests/test_pressure_point_flux/test_pressure_point_flux_line_element3D3N/ProjectParameters.json

```diff
@@ -0,0 +1,157 @@
+  },
+  "solver_settings" : {
+    "solver_type"                        : "Pw",
+    "model_part_name"                    : "PorousDomain",
+    "domain_size"                        : 3,
+    "model_import_settings" : {
+      "input_type"                       : "mdpa",
+      "input_filename"                   : "test_pressure_point_flux_line_element3D3N"
+    },
+    "material_import_settings" : {
+      "materials_filename"               : "../Common/MaterialParameters3D.json"
+    },
+    "time_stepping" : {
+      "time_step"                        : 3.600,
+      "max_delta_time_factor"            : 1000
+    },
+    "buffer_size"                        : 2,
+    "echo_level"                         : 0,
+    "clear_storage"                      : false,
+    "compute_reactions"                  : false,
+    "move_mesh_flag"                     : false,
+    "reform_dofs_at_each_step"           : false,
+    "nodal_smoothing"                    : false,
+    "block_builder"                      : true,
+    "solution_type"                      : "Transient_Groundwater_Flow",
+    "scheme_type"                        : "Backward_Euler",
+    "reset_displacements"                : false,
+    "newmark_beta"                       : 0.25,
+    "newmark_gamma"                      : 0.5,
+    "newmark_theta"                      : 0.5,
+    "rayleigh_m"                         : 0.0,
+    "rayleigh_k"                         : 0.0,
+    "strategy_type"                      : "newton_raphson",
+    "convergence_criterion"              : "water_pressure_criterion",
+    "displacement_relative_tolerance"    : 1.0E-4,
+    "displacement_absolute_tolerance"    : 1.0E-9,
+    "residual_relative_tolerance"        : 1.0E-4,
+    "residual_absolute_tolerance"        : 1.0E-9,
+    "water_pressure_relative_tolerance"  : 1.0E-4,
+    "water_pressure_absolute_tolerance"  : 1.0E-9,
+    "min_iterations"                     : 2,
+    "max_iterations"                     : 15,
+    "number_cycles"                      : 2,
+    "reduction_factor"                   : 0.5,
+    "increase_factor"                    : 2.0,
+    "desired_iterations"                 : 2,
+    "max_radius_factor"                  : 10.0,
+    "min_radius_factor"                  : 0.1,
+    "calculate_reactions"                : true,
+    "max_line_search_iterations"         : 5,
+    "first_alpha_value"                  : 0.5,
+    "second_alpha_value"                 : 1.0,
+    "min_alpha"                          : 0.1,
+    "max_alpha"                          : 2.0,
+    "line_search_tolerance"              : 0.5,
+    "rotation_dofs"                      : true,
+    "linear_solver_settings" : {
+      "solver_type"                      : "bicgstab",
+      "tolerance"                        : 1.0e-6,
+      "max_iteration"                    : 1000,
+      "scaling"                          : true,
+      "preconditioner_type"              : "ilu0"
+    },
```

Comment:
It may be more failsafe to use a direct solver for these tests with a limited number of D.O.F.